### ISSUE TITLE
Status tag contrast is too low

### ISSUE THREAD

**Issue by saevarom** [wagtail/wagtail#5778]
# Issue Summary

When working on #5370 I discovered that the secondary status tags shown in page lists, redirect lists, website lists and possibly other places are probably not WCAG compliant. Since these elements are not for decoration but to indicate information about the status of the items, they should have proper contrast ratios. This is fixed for now in #5776 where drafts are no longer using the secondary tag, but there are other places where they are used as mentioned before.

Redirects:
<img width="100" alt="Screenshot 2020-01-23 at 13 52 13" src="https://user-images.githubusercontent.com/143557/72991993-e3b2aa00-3dea-11ea-8f5d-243631b00223.png">

Sites:
<img width="111" alt="Screenshot 2020-01-23 at 13 53 39" src="https://user-images.githubusercontent.com/143557/72991994-e44b4080-3dea-11ea-963b-9e7bd4f00393.png">

Pages:
<img width="117" alt="Screenshot 2020-01-23 at 14 14 40" src="https://user-images.githubusercontent.com/143557/72991997-e44b4080-3dea-11ea-8262-d1c165bc50e1.png">

**Comment by thibaudcolas** [comment]
Thanks for opening this @saevarom. I’ve checked and the contrast ratio is indeed too low – WCAG 2.1 AA requires a contrast ratio of [3:1 for non-text elements](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). The status tag is below that both for its text (for which the target is much higher) and its border.

**Comment by thibaudcolas** [comment]
Re-testing this in 2024, I believe this has been addressed, with "draft" status tags using the same color as for live pages. Closing!
